psi4/src/psi4/libmints/wavefunction.cc

```diff
@@ -332,6 +340,11 @@ std::shared_ptr <Wavefunction> Wavefunction::c1_deep_copy(std::shared_ptr<BasisS
         wfn->arrays_[kv.first] = kv.second->clone();
     }
 
+    wfn->PCM_enabled_ = PCM_enabled_;
+    if (wfn->PCM_enabled_) {
+      wfn->PCM_ = std::make_shared<PCM>(PCM_.get());
+    }
```

Comment:
Is this actually a deepcopy?

Reply:
Yes, it calls the copy constructor `PCM::PCM(PCM * other)`